src/main/java/org/genomicsdb/importer/extensions/CallSetMapExtensions.java

```diff
@@ -221,39 +224,47 @@ default GenomicsDBCallsetsMapProto.CallsetMappingPB generateSortedCallSetMap(
      * @param inputSampleNameToPath    map from sample name to VCF/BCF file path
      * @param newCallsetMapPB callset mapping protobuf for new callsets
      * @return merged callsets for callsets to TileDB rows
-     * @throws ParseException when there is an error parsing callset jsons
+     * @throws JsonFormat.ParseException when there is an error parsing callset jsons
      */
     default GenomicsDBCallsetsMapProto.CallsetMappingPB mergeCallsetsForIncrementalImport(
             final String callsetMapJSONFilePath,
             final Map<String, URI> inputSampleNameToPath,
             final GenomicsDBCallsetsMapProto.CallsetMappingPB newCallsetMapPB) 
-            throws ParseException {
+            throws JsonFormat.ParseException {
         String existingCallsetsJSON = GenomicsDBUtils.readEntireFile(callsetMapJSONFilePath);
         GenomicsDBCallsetsMapProto.CallsetMappingPB.Builder callsetMapBuilder =
                 newCallsetMapPB.toBuilder();
         checkDuplicateCallsetsForIncrementalImport(existingCallsetsJSON, inputSampleNameToPath);
-        merge(existingCallsetsJSON, callsetMapBuilder);
+        try {
+          new JsonFormat().merge(new ByteArrayInputStream(existingCallsetsJSON.getBytes()), callsetMapBuilder);
+        } catch (IOException e) {
+          throw new GenomicsDBException(String.format("Could not merge incremental import's callsets with existing callsets : %s", e.getMessage()));
```

Comment:
Should the `GenomicsDBException` be added to the javadoc? I see that `checkDuplicateCallsetsForIncrementalImport` seems to follow that convention, but if that's not common in the library as a whole this comment can be ignored.

Reply:
Actually, even if it is not common in the library as a whole, we need to clean up javadoc as we go. So, yes, I will add the throws clause to javadoc. Thanks.

Reply:
Looks like `GenomicsDBException extends RuntimeException` which means a throws clause is not required in the javadoc. However, should we have a throws clause for all thrown `GenomicsDBException` for consistency sake? gatk does not have to explicitly look for GenomicsDBException as it is a RuntimeException, but if we start having a throws clause, it may be confusing to the gatk authors. What are your thoughts, @mlathara? Leave it as is?

Reply:
I think I may have mistakenly added `@throws GenomicsDBException` to:
importer/GenomicsDBImporter.java
importer/extensions/CallSetMapExtensions.java

If we remove those two instances, the rest is consistent in that they don't add that to the javadoc. My vote would be to do that.